functional-test/src/test/groovy/org/rundeck/tests/functional/api/job/JobExecutionSpec.groovy

```diff
@@ -731,45 +727,35 @@ class JobExecutionSpec extends BaseContainer {
         def execId = parsedExecutionsResponse.id
 
         then:
-        execId > 0
+        !execId?.isBlank()
```

Comment:
shouldn't the execId always be not null?
```suggestion
        !execId.isBlank()
```

Reply:
this applies to every execId validation

Reply:
Nice catch, this would made the test pass if execId is null. Fixed.